protocol/lib/metrics/metric_keys.go

```diff
@@ -0,0 +1,39 @@
+package metrics
+
+// Metrics Keys Guidelines
+// 1. Be wary of length
+// 2. Prefix by module
+// 3. Suffix keys with a unit of measurement
+// 4. Delimit with '_'
+// 5. Information such as callback type should be added as tags, not in key names.
+
+// Clob Metrics Keys
+const (
+	// Stats
```

Comment:
Nit question but is there a benefit to formatting these `consts` by metrics type? Seems unnecessary IMO

Reply:
It's useful for when I migrate files over one by one since I can bucket the types, can't hurt to have it here for now. If you feel strongly about this I can combine later.